Exercise_3.java

```diff
@@ -14,12 +14,20 @@
         } 
     } 
   
+    //Approach : Applying Floyd's slow and fast pointer method
     /* Function to print middle of linked list */
    //Complete this function
     void printMiddle() 
     { 
         //Write your code here
 	//Implement using Fast and slow pointers
+    	Node slow = head, fast = head;
+    	while(fast != null && fast.next != null)

```

Comment:
This condition will give you a false result when there are only 2 elements in the linked list. Try making it while(fast.next != null && fast.next.next != null)